### ISSUE TITLE
RW model schema cleaning

### ISSUE THREAD

**Issue by eosrei** [roundware/roundware-server#165]
Is the EventType model just clutter? https://github.com/hburgund/roundware-server/blob/64360681fe1943bc3eca5acbeaac0d2da43b5d1a/roundware/rw/models.py#L291

It's empty on the production server:
```
mysql> select * from rw_eventtype;
Empty set (0.00 sec)
```

**Comment by eosrei** [comment]
Just encountered: https://code.djangoproject.com/ticket/17946 for Tag fixtures. Yay!
```
django.db.utils.IntegrityError: Problem installing fixture '/vagrant/roundware/rw/fixtures/sample_project.json': Could not load rw.Tag(pk=3): (1062, "Duplicate entry '3-3' for key 'from_tag_id'")
```
Will be applying: https://github.com/django/django/commit/9fecb86a5221982ea753270b7d2b4c6487f79941 via MonkeyPatch.

**Comment by eosrei** [comment]
FYI `asset.initialenvelope_id` is used. This is the error you get if it is removed:
```
CommandError: System check identified some issues:

ERRORS:
<class 'roundware.rw.admin.AssetInline'>: (admin.E202) 'rw.Asset' has no ForeignKey to 'rw.Envelope'.
```

**Comment by eosrei** [comment]
select_method and ui_mode are now readable within the SQL.
```
mysql> select * from rw_masterui;
+----+----------------------------------------+-----------------+--------+-------+------------+---------------+---------+
| id | name                                   | tag_category_id | active | index | project_id | select_method | ui_mode |
+----+----------------------------------------+-----------------+--------+-------+------------+---------------+---------+
|  1 | What genders do you want to listen to? |               3 |      1 |     1 |          1 | min_one       | listen  |
|  3 | What do you want to listen to?         |               2 |      1 |     3 |          1 | min_one       | listen  |
|  4 | What gender are you?                   |               3 |      1 |     1 |          1 | single        | speak   |
|  6 | What do you want to talk about?        |               2 |      1 |     3 |          1 | single        | speak   |
|  7 | Who do you want to hear?               |               4 |      1 |     2 |          1 | min_one       | listen  |
|  8 | Who are you?                           |               4 |      1 |     2 |          1 | single        | speak   |
+----+----------------------------------------+-----------------+--------+-------+------------+---------------+---------+
6 rows in set (0.00 sec)
```
 @hburgund This may result in minor changes to the webviews?

**Comment by hburgund** [comment]
If this causes any changes in the ```get_tags``` JSON response, then yes, it will most likely require some minor webview changes to be made.  But I assume you'll be updating the server code to incorporate this change in that JSON response, yes?  I guess there may still be a difference to be managed given that the names of the select methods have changed.  I know how you like short names ;-)  ```ui_mode``` is the same, so that shouldn't cause any problems.

**Comment by eosrei** [comment]
I actually made it longer. `select_method` was my first custom migration and I didn't realize I could implement without changing the name. If that's the only issue, I can fix that commit. The output of the select methods in this JSON should be the same.

**Comment by hburgund** [comment]
I was actually talking about the values, not the keys, but both are important, of course:
```
"select": "multi_at_least_one"
```
This is how it is included now in the JSON response, but I'm assuming with your changes, it would be:
```
"select": "min_one"
```
This is fine, but as you say, I would just have to update the webviews to handle the new value since I believe they behave differently based on this value.  This is easy, though, so if you think it's worth changing the ```select_method``` values, that's fine with me.

**Comment by eosrei** [comment]
Fixed by #200
